app/assets/stylesheets/local/show.scss

```diff
@@ -1,29 +1,165 @@
-// There seems to be no existing css class allowing us to scope
-// this stuff just to generic work 'show' page. If there was, I'd use it.
-// Perhaps it should be added later.
 
-.additional-titles {
-  h2 {
-    @extend h4;
+
+.show-page-image {
+  position: relative;
+
+  .show-page-image-image {
+    width: 100%;
+    // keep crazy long aspect ratio images from being
+    // so long, although there will be some distortion.
+    max-height: 80vw;
+  }
+}
+
+.show-page-image-viewer-link {
+  display: block;
+  position: absolute;
```

Comment:
can you comment the display / position settings here? Sometimes I find it helpful to have a little extra context on these since they are often about setting up behavior in child elements.